server/src/main/java/org/opensearch/gateway/ReplicaShardAllocator.java

```diff
@@ -441,9 +470,19 @@ private MatchingNodes findMatchingNodes(
         return new MatchingNodes(matchingNodes, nodeDecisions);
     }
 
+    private Map<DiscoveryNode, StoreFilesMetadata> adaptToNodeShardStores(AsyncShardFetch.FetchResult<NodeStoreFilesMetadata> data) {
```

Comment:
Consider naming the method to `convertToNodeShardStores`

Reply:
I think convert, adapt, transform anything can be used for the above function, do you think one is better for some reason?

Reply:
`adapt` implies changing the state on basis of outside signals, but here we are only converting a data object into another type. `transform` is a better fit.